server/src/test/java/org/bonitasoft/web/rest/server/BonitaRestletApplicationTest.java

```diff
@@ -40,13 +41,26 @@ public void should_application_register_bdm_resources() throws Exception {
     @Test
     public void should_application_register_extension_resources() throws Exception {
         //given
-        final BonitaRestletApplication bonitaSPRestletApplication = new BonitaRestletApplication(finderFactory, bonitaJacksonConverter);
+        final BonitaRestletApplication bonitaRestletApplication = new BonitaRestletApplication(finderFactory, bonitaJacksonConverter);
 
         //when
-        bonitaSPRestletApplication.buildRouter();
+        bonitaRestletApplication.buildRouter();
 
         //then
         Mockito.verify(finderFactory).createExtensionResource();
     }
 
+
+    @Test
+    public void should_application_register_i18n_resources() throws Exception {
```

Comment:
"Should application register i18n resources" is not a well formed english sentence